test_e2e.sh

```diff
@@ -285,7 +285,7 @@ assert_file_exists "$ROOT"/link/file
 assert_file_eq "$ROOT"/link/file "$TESTCASE 1"
 # Wrap up
 remove_sync_container
-wait
+interval
```

Comment:
These changes are wrong - `wait` here is a shell command.

Reply:
I apologize for my mistake. 
I reverted command.